drivers/misc/renesas_rx_lvd/Kconfig

```
@@ -0,0 +1,19 @@
+# Copyright (c) 2025 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Renesas RX Voltage Detection Circuit
+
+config RENESAS_RX_LVD
+	bool "Renesas RX Low-Voltage Detection Driver"
+	depends on DT_HAS_RENESAS_RX_LVD_ENABLED
+	default y
+	select USE_RX_RDP_LVD
+	select PINCTRL
+	help
+	  Enable Renesas RX low voltage detection driver
+
+config RENESAS_RX_LVD_INIT_PRIORITY
+	int "RX LVD initialization priority"
+	default KERNEL_INIT_PRIORITY_DEVICE
+	help
+	  System initialization priority for RX LVD driver.
```

Should be wrapped by `RENESAS_RX_LVD`
```suggestion
if RENESAS_RX_LVD

config RENESAS_RX_LVD_INIT_PRIORITY
	int "RX LVD initialization priority"
	default KERNEL_INIT_PRIORITY_DEVICE
	help
	  System initialization priority for RX LVD driver.

endif # RENESAS_RX_LVD
```